x-pack/plugins/observability/public/components/slo/slo_status_badge/slo_group_by_badge.tsx

```diff
@@ -12,17 +12,18 @@ import { euiLightVars } from '@kbn/ui-theme';
 import React from 'react';
 
 export interface Props {
+  cardView?: boolean;
   slo: SLOWithSummaryResponse;
 }
 
-export function SloGroupByBadge({ slo }: Props) {
+export function SloGroupByBadge({ slo, cardView }: Props) {
   if (!slo.groupBy || slo.groupBy === ALL_VALUE) {
     return null;
   }
 
   return (
     <EuiFlexItem grow={false}>
-      <EuiBadge color={euiLightVars.euiColorDisabled}>
+      <EuiBadge color={cardView ? 'default' : euiLightVars.euiColorDisabled}>
```

Comment:
@shahzad31 I think this check is unnecessary, since you are not showing the group_by badge in the compact view, no?